src/ONYXKEYS.ts

```diff
@@ -409,7 +409,7 @@ type OnyxFormValuesMapping = {
     [ONYXKEYS.FORMS.HOME_ADDRESS_FORM]: FormTypes.Form;
     [ONYXKEYS.FORMS.NEW_ROOM_FORM]: FormTypes.NewRoomForm;
     [ONYXKEYS.FORMS.ROOM_SETTINGS_FORM]: FormTypes.Form;
-    [ONYXKEYS.FORMS.NEW_TASK_FORM]: FormTypes.Form;
```

Comment:
Merge the newest main, this is already changed on main

Reply:
merged